src/test/resources/unit/spdx-maven-plugin-test/pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>org.spdx</groupId>
-  <artifactId>spdx maven plugin test</artifactId>
+  <artifactId>spdx-maven-plugin-test</artifactId>
```

Comment:
Thanks to you for the quick review!

Good point here. I've URL encoded the artifactId, which replaces the whitespaces with +. This will also encode any other symbol not valid in a PURL.